modular_zubbers/code/modules/quirks/code/negative_quirks/well_trained.dm

```diff
@@ -0,0 +1,84 @@
+/datum/quirk/well_trained
+	name = "Well-Trained"
+	desc = "You absolutely love being dominated. The thought of someone with a stronger character than yours is enough to make you act up."
+	icon = "sort-down"
+	medical_record_text = "Patient can be easily swayed by a sufficiently assertive individual"
+	// Yes, it should be neutral. Yes, this is a bad idea. This is funny and multiple people are saying it's time to be funny.
+	value = -1
+	gain_text = "<span class='notice'>You feel like being someone's pet</span>"
+	lose_text = "<span class='notice'>You no longer feel like being a pet...</span>"
+	quirk_flags = QUIRK_HUMAN_ONLY | QUIRK_HIDE_FROM_SCAN | QUIRK_PROCESSES
+	erp_quirk = TRUE
+	var/mood_category = "dom_trained"
+	var/mob/living/carbon/human/last_dom
+
+/datum/quirk/well_trained/add(client/client_source)
+	. = ..()
+	RegisterSignal(quirk_holder, COMSIG_MOB_EXAMINING, PROC_REF(on_dom_examine))
+
+/datum/quirk/well_trained/remove()
+	. = ..()
+	UnregisterSignal(quirk_holder, COMSIG_MOB_EXAMINING)
+
+/datum/quirk/well_trained/proc/on_dom_examine(atom/source, mob/living/user, list/examine_list)
```

Comment:
this will all happen so fast. how about making the initial stuff take a moment to run. right before the dead check.